src/app/app-routing.module.ts

```diff
@@ -1,10 +1,15 @@
-import { NgModule } from '@angular/core';
-import { RouterModule, Routes } from '@angular/router';
+import { CurrencyHomeComponent } from "./widgets/exchange/currency-home/currency-home.component";
+import { NgModule } from "@angular/core";
+import { RouterModule, Routes } from "@angular/router";
+import { CurrencyAdminComponent } from "./widgets/exchange/currency-admin/currency-admin.component";
 
-const routes: Routes = [];
+const routes: Routes = [
+  { path: "", component: CurrencyHomeComponent },
+  { path: "currency/admin", component: CurrencyAdminComponent },
```

Comment:
One good practice for not having typos one the routing strings is to create enums ;)